drivers/SmartThings/philips-hue/src/disco.lua

```diff
@@ -33,7 +34,12 @@ process_discovered_light
 ---@param _ table
 ---@param should_continue function
 function HueDiscovery.discover(driver, _, should_continue)
+  if HueDiscovery.discovery_active then
+    log.info("Hue discovery already in progress, ignoring new discovery request")
```

Comment:
should there be an early return here?

Reply:
Oof yes. I had one, then I reset the entire branch cuz I accidentally based it off #931, and it didn't get brought over.